util/plancodec/codec.go

```diff
@@ -316,6 +313,28 @@ func encodeID(planType string, id int) string {
 	return strconv.Itoa(planID) + idSeparator + strconv.Itoa(id)
 }
 
+func encodeTaskType(isRoot bool, storeType kv.StoreType) string {
```

Comment:
can we add unit tests for encode and decode TaskType ?

Reply:
Comment addressed. PTAL~ @hanfei1991